pkg/database/sql/writer.go

```diff
@@ -7,52 +7,12 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"time"
 
 	"github.com/kubearchive/kubearchive/pkg/database/sql/facade"
 	"github.com/kubearchive/kubearchive/pkg/models"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-func (db *sqlDatabaseImpl) WriteResource(ctx context.Context, k8sObj *unstructured.Unstructured, data []byte, lastUpdated time.Time) error {
```

Comment:
As we are deleting this method, then the `sqlDatabaseImpl` partially implements `Database` interface, so we should update our [docs](https://kubearchive.github.io/kubearchive/main/integrations/database.html#_adding_a_new_database_engine) explicitly saying that `sqlDatabaseImpl` is a partial implementation of `Database` so the developer should make sure that the new engine implementation fully implements all the methods of the `Database` interface..

Reply:
Added